style/main.css

```diff
@@ -194,6 +194,30 @@ form[role="dialog"][data-type="confirm"].lcdui-alert section {
   left: calc(50% - 17px);
 }
 
+.background-screen {
+  z-index: 2;
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: auto;
+  bottom: auto;
+  background-color: white;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-image: url("../img/icon-128.png");
```

Comment:
Unsure why, but on a Flame running FxOS 2.1, this background image doesn't appear in the screenshot of the app on the Recent Apps screen. All I see in the screenshot is a blank white background. Once I switch to the app, however, I do see the image (for a moment, until the splash screen appears).